docs/gallery-submission.md

```diff
@@ -4,175 +4,216 @@ title: Connector Gallery Submission Guide
+Include a copy of the manifest file (.taco) for your connector. Information in this file should match the information submitted for the items below. TACO files must be [signed by a certificate authority](https://tableau.github.io/connector-plugin-sdk/docs/log-entries) to be accepted.
 
-### TDVT results &amp; explanations of failures
+##### TDVT Results and Explanations of Failures
 
 TDVT will produce a number important files required for a review:
 
-- test\_results\_combined.csv (note: CSV is preferred)
+- test\_results\_combined.csv (CSV is preferred)
 - tdvt\_log\_combined.txt
 - tdvt\_actuals\_combined.txt
 - tdvt\_output\_combined.txt
 - tabquery\_logs.zip
 
-TDVT should be passing more than 90% of test cases in order to be accepted for the majority of cases. A summary of failed test cases should be provided for review.
+In most cases, TDVT should be passing more than 90% of test cases to be accepted. A summary of failed test cases should be provided for review.
+
+##### Connector Name
 
-### Connector Name
+Include the name for your application/database to which the connector connects .
 
-The name for your application/database that the connector connects to.
+##### Short Description
 
-### Short Description
+Briefly describe details about your connector that are most relevant for customers. This displays on the main Connectors page of the Extensions Gallery. Maximum 116 characters. Plain text only.
 
-A short blurb about your connector which is most relevant for customers, only shown on the front page. Maximum 116 characters. Plain text only.
+##### Long Description
 
-### Long Description
+For your connector's detail page, provide a long description. Include these three parts.
 
-Long descriptions have 3 parts.
+- A description of your application or database.
+- A description of what the connector does that makes better than using an Other ODBC/JDBC connection.  
 
-1. A description of your application or database.
-2. A description of what the connector does that makes better than using an Other ODBC/JDBC connection
-  1. Example may be: This connector provides a smooth and easy way to connect to Google Big Query. By using this connector, customers will have the ability to leverage Google's internal storage and extract APIs which offer up to 10x performance increases when extracting large amounts of data. This connector also allows users to use custom OAuth which is not currently supported natively with an Other JDBC connection.
-3. Known issues or version requirements for the connector
+   Example:   
+   
+   _This connector provides a smooth and easy way to connect to Google Big Query. By using this connector, customers can leverage Google's internal storage and extract APIs which offer up to 10x performance increases when extracting large amounts of data. This connector also allows users to use custom OAuth which is not currently supported natively with an Other JDBC connection._
```

Comment:
Maybe a placeholder name instead of an actual company's product (Google Big Query).